src/gui/guimain.cpp

```diff
@@ -128,14 +127,23 @@ extern "C" void gui_run(void) {
     //set loop callback (will be called every time inside gui_loop)
     gui_loop_cb = _gui_loop_cb;
     //int8_t gui_timeout_id;
+    GuiDefaults::menu_timeout_enabled = variant_get_ui8(eeprom_get_var(EEVAR_MENU_TIMEOUT));
     while (1) {
         Screens::Access()->Loop();
         // show warning dialog on safety timer expiration
         if (marlin_event_clr(MARLIN_EVT_SafetyTimerExpired)) {
             MsgBoxInfo(_("Heating disabled due to 30 minutes of inactivity."), Responses_Ok);
         }
         gui_loop();
-        /*if (menu_timeout_enabled) {
+        /*if (marlin_message_received()) {
```

Comment:
what is this commented code?

Reply:
This is old pop up msg & menu timeout logic before GUI refactoring - new implementation is prepared to review